docs/configuration.md

```diff
@@ -0,0 +1,68 @@
+# Run-time configuration
+
+RedisTimeSeries supports a few run-time configuration options that should be determined when loading the module. In time more options will be added. 
+
+## Passing Configuration Options During Loading
+
+In general, passing configuration options is done by appending arguments after the `--loadmodule` argument in the command line, `loadmodule` configuration directive in a Redis config file, or the `MODULE LOAD` command. For example:
+
+In redis.conf:
+
+```
+loadmodule redistimeseries.so OPT1 OPT2
+```
+
+From redis-cli:
+
+```
+127.0.0.6379> MODULE load redistimeseries.so OPT1 OPT2
+```
+
+From command line:
+
+```
+$ redis-server --loadmodule ./redistimeseries.so OPT1 OPT2
+```
+
+## RedisTimeSeries configuration options
+
+### COMPACTION_POLICY {policy}
+
+Default compaction/downsampling rules for newly created key with `TS.ADD`.
+
+Each rule is seperated by a semicolon (`;`), the rule consists of several fields that are separated by a colon (`:`):
+* aggregation function - avg, sum, min, max, count, first, last
+* time bucket - number and the time representation (Example for 1 minute: 1M)
+    * m - millisecond
+    * M - minute
+    * s - seconds
+    * d - day
+* retention time - in milliseconds
```

Comment:
hello, It looks like your markdown parser did not rendered correctly this part of the documentation.
https://oss.redislabs.com/redistimeseries/configuration/#compaction_policy-policy

I wanted to fixed it, but I'm not familiar with ruby toolchain. Sorry